source/MRMesh/MRMeshSave.cpp

```diff
@@ -221,6 +227,61 @@ VoidOrErrStr toBinaryStl( const Mesh & mesh, std::ostream & out, ProgressCallbac
     return {};
 }
 
+VoidOrErrStr toAsciiStl( const Mesh& mesh, const std::filesystem::path& file, ProgressCallback callback )
+{
+    std::ofstream out( file, std::ofstream::binary );
+    if ( !out )
+        return unexpected( std::string( "Cannot open file for writing " ) + utf8string( file ) );
+
+    return toAsciiStl( mesh, out, callback );
+}
+
+VoidOrErrStr toAsciiStl( const Mesh& mesh, std::ostream& out, ProgressCallback callback )
+{
+    MR_TIMER;
+#define SOLID_NAME "mesh"
```

Comment:
Can we avoid #defines? And please write something like "solid MeshInspector.com" (we need publicity)